docs/src/get-started/goals.rst

```diff
@@ -0,0 +1,98 @@
+representation, new models, and advanced research technics. Unfortunately each
+one of these libraries lives mostly separated from the others, resulting in a
+lot of duplicated effort. With equistore, we want to provide a way for these
+libraries to communicate with one another, by giving everyone a *lingua franca*,
+a way to share data and metadata.
+
+This goal is enabled by multiple features of equistore: first, equistore allows
+storing data coming from many different sources, without requiring to first
+convert the data to a specific format. Currently, we support data stored inside
+numpy arrays, torch tensor (including tensors on GPU or other accelerators), as
+well as arbitrary user-defined C, C++, and Rust array types. A second part of
+this goal is achieved by also storing metadata together with the data,
+communicating between libraries exactly **what** is stored in the different
+arrays. We also store both data and gradients of this data with respect to
+arbitrary parameters together, enabling for example training of models using
+energy, forces and virial. Finally, we also make sure that the data storage is
+as efficient as possible and can exploit the inherent sparsity of atomistic
+data, in particular in gradients.
+
+As a developer a library in the atomistic machine learning ecosystem, you can
+provide conversion functions to and from equistore
+:py:class:`equistore.TensorMap` to enable using your library in conjunction with
+the rest of the equistore-compatible libraries!
+
+.. TODO: add illustration
+
+--------------------------------------------------------------------------------
+
+The following libraries use equistore either as input, output or both:
+
+- `equisolve <https://github.com/lab-cosmo/equisolve/>`_: a companion to
+  equistore implementing common ML models that take all the data as equistore
+  TensorMaps;
+- `rascaline <https://github.com/Luthaf/rascaline/>`_: a library computing
+  physics-inspired representations of atomic systems. Rascaline always outputs
+  its representations in equistore format;
+- `Q-stack <https://github.com/lcmd-epfl/Q-stack/>`_: library of pre- and
+  post-processing tasks for Quantum Machine Learning; can output some of its
+  data in equistore format;
+
+
+Equistore as a prototyping tool
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The second objective of equistore is to provide functionalities to be a
+prototyping tool for new models. While it is possible to use equistore to only
+exchange data between libraries (and immediately convert everything to library
+specific format); we also provide tools to operate on equistore data, staying in
+the equistore format.
+
+We call these tools :ref:`operations <python-api-operations>`, and they
+available in the Python interface to equistore. By using combining multiple
+operations, you can build custom machine learning models, using data and
+representations coming from arbitrary equistore-compatible libraires in the
+ecosystem.
```

Comment:
It's unclear the benefit of keeping things in equistore format, rather than exporting as numpy/torch and running your normal operations